src/pages/help-centre/_article.tsx

```diff
@@ -36,6 +36,11 @@ const TabWrapper = styled.div`
 const ContactContainer = styled.div`
     margin-top: 8rem;
 `
+
+const queryParams = new URLSearchParams(window.location.search)
+const platform_name = queryParams.get('platform')
+const platform_list = ['derivgo', 'p2p']
+const platform = platform_list.includes(platform_name) ? platform_name : ''
```

Comment:
We can make this a utility since we use it in multiple places.

Reply:
Yep, it would be good. Thank you